readme.md

```diff
@@ -764,6 +764,7 @@
 - [Plotters](https://github.com/beardicus/awesome-plotters#readme) - Computer-controlled drawing machines and other visual art robots.
 - [Robotic Tooling](https://github.com/protontypes/awesome-robotic-tooling#readme) - Free and open tools for professional robotic development.
 - [LIDAR](https://github.com/szenergy/awesome-lidar#readme) - Sensor for measuring distances by illuminating the target with laser light.
+- [Open Hardware](https://github.com/delftopenhardware/awesome-open-hardware#readme) - physical artifacts whose designs are shared publicly so that others can make, modify, distribute and use those things.
```

Comment:
* Please start description with uppercase.
* Not obvious email for "email me privately" in Contributing.

Reply:
Innovation for cloud storage security

Reply:
> * Not obvious email for "email me privately" in Contributing.  

Fixed this!

Reply:
> Innovation for cloud storage security

Not sure what to do with this..